### DISCUSSION TITLE
A Vuetify version for Svelte?

### DISCUSSION THREAD

**Discussion by frederikhors** [vuetifyjs/vuetify#12139] [General]
### Problem to solve
I couldn't find anything looking for "Svelte" in the issues.

### Proposed solution

This is why I ask aloud: will there ever be a version of Vuetify for Svelte?

**Comment by MajesticPotatoe** [comment]
Vuetify requires Vue to run. The likelihood of us generating Vuetify for another framework is pretty low. Not to say that someone out there decides they want to take it and generate it for Svelte.
